### ISSUE TITLE
roots-collections crashes on the latest roots

### ISSUE THREAD

**Issue by orenmizr** [netlify/roots-collections#1]
Potentially unhandled rejection [86] TypeError: f.content.match is not a function among other related errors. 

the instructions are easy. to make sure i use it correctly i also downloaded netify cms example which also throws errors on collections. 

can you please check it out?

**Comment by biilmann** [comment]
Which version of roots is that?

I've just tested locally with roots 3.1.0 and the extension should work (though I found two unrelated issues that I just pushed a fix for).

Cloning the netlify CMS example and running `npm install` and then `roots compile` with roots 3.1.0 also works fine here.

**Comment by orenmizr** [comment]
i followed instructions to the letter. my app.coffee uploaded here: https://github.com/orenmizr/roots_tryout

in my example for instance - post does not have .body - it got the frontmatter plus content which is the entire html with doctype,

i am running latest roots. 

**update**: also tried with roots-posts extension. got back ```Potentially unhandled rejection [43] TypeError: f.content.match is not a function

**Comment by orenmizr** [comment]
**update**: removed all node-modules - re-installed roots globally. now the error:  

```
No layout views/post.jade found for collection_posts (WARNING: non-Error used)
```

```
  console.log(@layoutFile)
  throw("No layout #{@layoutFilename} found for #{category}") unless @layoutFile
```

it appears that my @layoutFile is undefined. 
i am sending this options via app.coffee: { folder: 'posts', layout: 'post' }
Ii have posts folder with markdown.
i have views/ post.jade that is a block extension to layout. 

all appears normal...
- i can't seem to run "root watch" via debuggers like webstorm or iron node. if you know how - please tell me.
- i don't why it shoots that error. by this point i've deleted most of my content and put netify/roots example content instead. 

hopefully someone may be able to help #hint

**Comment by biilmann** [comment]
This turned out to be bad path handling on windows. Fixed in `0.3.2`
